website/index.html

```diff
@@ -0,0 +1,440 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <title>Indiepen</title>
+    <meta
+      name="description"
+      content="An independent and privacy-friendly solution to present your code example to the people."
+    />
+    <meta name="HandheldFriendly" content="True" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="/website/favicon.png" />
+    <link
+      rel="stylesheet"
+      href="/website/styles.css"
+      media="none"
+      onload="if(media!='all')media='all'"
+    />
+    <noscript>
+      <link rel="stylesheet" href="/website/styles.css" />
+    </noscript>
+
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      rel="preload"
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;900&family=Source+Code+Pro:wght@500&display=swap"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;900&family=Source+Code+Pro:wght@500&display=swap"
+      media="print"
+      onload="this.media='all'"
+      rel="stylesheet"
+    />
+    <link rel="canonical" href="https://indiepen.tech/" />
+    <meta name="referrer" content="no-referrer-when-downgrade" />
+    <meta property="og:site_name" content="Indiepen" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Indiepen" />
+    <meta
+      property="og:description"
+      content="An independent and privacy-friendly solution to present your code example to the people."
+    />
+    <meta property="og:url" content="https://indiepen.tech/" />
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:title" content="Indiepen" />
+    <meta
+      name="twitter:description"
+      content="An independent and privacy-friendly solution to present your code example to the people."
+    />
+    <meta name="twitter:url" content="https://indiepen.tech/" />
+    <style>
```

Comment:
Just out of curiosity: Why do we have inline styling and the `styles.css` file?

Reply:
Whoops, totally forgot about the inline stuff. But in general it's a good thing for performance. We load all the styles for above the fold and then we load the `styles.css` asynchronously by:

```html
    <link
      rel="stylesheet"
      href="/website/styles.css"
      media="none"
      onload="if(media!='all')media='all'"
    />
```

---

So this import is non-blocking and can be loaded later because it will not cause a flash of content. But I guess that can put the inline styles into a seperate stylesheet and load it like:

```html
    <link
      rel="stylesheet"
      href="/website/blocking-styles.css"
      media="screen"
    />
```

But this would mean that we have another request and this would be benefitial for a bigger site with multiple pages, beacause you only load the stylesheet once. But in our case it's still fast to put the "critical" styling into a `<style>`.

Reply:
Sounds reasonable :)